### PR TITLE
add org.coolercontrol.CoolerControl

### PR DESCRIPTION
**Pull request by codifryed**
### Please confirm your submission meets all the criteria

Please describe your application briefly below.

CoolerControl features a GUI for viewing all your system's sensors and for creating custom fan and
pump profiles based on any available temperature sensor. Paired with this GUI package is a systemd service that
controls all your devices in the background.
It's an extension of [liquidctl](https://github.com/liquidctl/liquidctl) and
[hwmon](https://hwmon.wiki.kernel.org/) with a focus on controlling cooling devices such as AIO
coolers and fans under Linux.

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
You didn't answer:

> Please describe your application briefly below.

File: org.coolercontrol.CoolerControl.yml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+separate-locales: false
+command: coolercontrol
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.rust-stable
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```
Comment (hfiguiere) [inline org.coolercontrol.CoolerControl.yml:14]: so there is no permission to display anything at all.

File: metainfo.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
+diff --git a/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml b/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml
+index b16da4e..5f298f8 100644
+--- a/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml
++++ b/packaging/metadata/org.coolercontrol.CoolerControl.metainfo.xml
+@@ -20,6 +20,10 @@
+         <p>
+             It's an extension of liquidctl and hwmon with a focus on controlling cooling devices such as AIO coolers and fans.
+         </p>
++        <p>
++            Note: The coolercontrold system package is required to control and communicate with your hardware devices.
```
Comment (hfiguiere) [inline metainfo.patch:10]: this is gonna work great out of the box.

File: org.coolercontrol.CoolerControl.yml (hunk 1 above)
Comment (codifryed) [reply org.coolercontrol.CoolerControl.yml:14]: The permissions are sufficient for the application to work and the X11-fallback works testing locally, due to the Tauri bug with wayland. Do you mean that there's something missing here?

File: metainfo.patch (hunk 2 above)
Comment (codifryed) [reply metainfo.patch:10]: There are several other Flatpaks that work like this but there is also a history here.
This project went through a name and implementation change and was previously called Coolero and there is an outdated flatpak here: https://flathub.org/apps/org.coolero.Coolero

This project now consists of several packages which is why this note is here and the idea is essentially to replace the above flatpak with this alternative for the current implementation. 

If this is not appropriate or there is an issue with this, please say so. The GUI also starts and displays some helpful dialog about the needed systemd service if it isn't already running.

**Comment by codifryed** [comment]
> You didn't answer:
> 
> > Please describe your application briefly below.

whoops, sorry about that. Updated.

File: org.coolercontrol.CoolerControl.yml (hunk 1 above)
Comment (hfiguiere) [reply org.coolercontrol.CoolerControl.yml:14]:
```suggestion
  - --socket=x11
```

the linter will prevent it anyway.

File: metainfo.patch (hunk 2 above)
Comment (hfiguiere) [reply metainfo.patch:10]: two wrong don't make a right.

**Pull request closed without merging**